docs/changelog.md

```diff
@@ -26,6 +26,46 @@
 .. role:: small
 ```
 
+## 2024-08-03 {small}`lamindb 0.75.1`
+
+Features.
+
+- 🚸 In Vitessce integration, enable a Vitessce button next to a dataset, also, document valid suffixes [PR](https://github.com/laminlabs/lamindb/pull/1799) [@falexwolf](https://github.com/falexwolf)
+- ✨ Allow exclude values in Curate.validate [PR](https://github.com/laminlabs/lamindb/pull/1798) [@sunnyosun](https://github.com/sunnyosun)
+
+User experience.
+
+- 🔇 Remove warning of loading non-default source [PR](https://github.com/laminlabs/bionty/pull/32) [@sunnyosun](https://github.com/sunnyosun)
+
+Docs.
+
+- 📝 Improve curate docs [PR](https://github.com/laminlabs/lamindb/pull/1795) [@Zethson](https://github.com/Zethson)
+
+Use cases.
+
+- 📝 Mutate tiledbsoma store [PR](https://github.com/laminlabs/lamin-usecases/pull/137) [@Koncopd](https://github.com/Koncopd)
```

Comment:
This was already linked in the "Guide" link in the last release. Please remove here.